packages/console/src/pages/ApplicationDetails/ApplicationDetailsContent/Branding/index.module.scss

```diff
@@ -6,7 +6,7 @@
   > * {
     flex: 1;
 
-    &:last-child {
+    &:nth-child(2) {
```

Comment:
why `nth-child`? if it's necessary, should we add comments?

Reply:
Could have two logo uploaders in a row when dark mode is enabled.  Comments added